conduct.rst

```diff
@@ -0,0 +1,252 @@
+When a long explanation is necessary,
+consider adding a summary.
+Try to bring new arguments to a conversation
+so that each message adds something unique to the thread,
+keeping in mind that the rest of the thread
+sill contains the other messages that have already been made.
+
+Try to stay on topic,
+especially in discussions that are already fairly large.
+If a discussion brings to mind some related
+but separate issue,
+please start a separate discussion for it.
+Everyone that is participating in an existing discussion
+will naturally wish to respond to your comment.
+If it is a separate discussion,
+each person can choose to join that discussion
+or spend their time some other way.
+
+Giving and receiving feedback
+=============================
+
+Give constructive, not critical feedback.
+Feedback is negatively critical
+when it surfaces something wrong with someone or something they produced,
+especially without any mention
+of ways to make their behavior or their product better.
+Critical feedback on work often looks like
+"you don't write enough tests"
+or "your code quality isn't good enough."
+Personal criticism can be more severe
+and often looks like "you should be less judgmental"
+or "you are a burden because you ask too many questions."
+Constructive feedback
+is more about how a person can do better
+rather than what they are doing wrong.
+If you want someone to do something better,
+you should tell them what better looks like.
+Ask a question to get a discussion rolling,
+to gain context,
+and then if you see room for improvement
+give declarative feedback to that effect.
+This creates an environment
+where people understand what success looks like
+instead of just feeling like they are unsuccessful.
+
+Code, configurations, and their reviews
+are also mechanisms for communication.
+Just as you shouldn't interact with people poorly in person,
+do not interact poorly through code or code review.
+
+You are not your products.
+Technical critiques are integral,
+and should be hard on the product,
+not on the producer.
+While it is important to care about your work
+and to produce the best thing you can,
+this can make review difficult.
+It is important to realize that it's better
+to find errors in review
+than in production
+and recognize that your work fits into a larger whole.
+
+Go about your review
+under the assumption
+that the decisions were made for a reason,
+not in a vacuum.
+Ask about circumstances if you're confused.
+
+Be pragmatic,
+ask for context,
+don't filibuster,
+don't block on style not explicitly covered in our style guides.
+
+Code, configurations, architecture, platforms, frameworks will need to be changed.
+Fight for your way if you think it's right, but not only to be right.
+
+Enactment
+=========
+
+If somebody requests that you stop a certain behavior
+(even if said behavior is not explicitly covered in this document),
+you are expected to comply immediately.
```

Comment:
To sort of emphasize the content of my previous couple comments, i'd insert something similar here... perhaps:

However, complying is not an admission of guilt or of a transgression.  Instead, it is an acknowledgement that further discussion of what caused the request is probably a good idea.  The comment could have been mis-stated or mis-read, it is probably a good idea work with the other party to fix the situation.